### PR TITLE
Updated Dungeon.java

### PR DESCRIPTION
**Pull request by anguyenq**
Added switch using "wasd" to moveHero()

### CODE REVIEW

**Comment by cwatss** [comment]
> My code is unfinished. We have not decided on how this class interacts with Dungeon.java so I left the key items (entrance, exit, crown1, crown2) in limbo for now.

Ok cool,  no worries I was just clarifying🙂👍

File: Room.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+		}
+	}
+	
+	/**
+	 * Returns a String representation of this instance.
+	 * 
+	 * @return a String representation of this instance.
+	 */
+	@Override
+	public String toString() {
```
Comment (cwatss) [inline Room.java:142]: To add to this method:
Name
HitPoints
Total Healing Potions
Total Vision Potions
Total Crown Pieces Found

**Comment by anguyenq** [comment]
To clarify the StringBuilder, it was used mainly a placeholder in case we wanted to increase the toString's capability, e.g. printing out a visual representation of the room with its doors.

Also, most of my code is based off the example program Tom shared in his assignment video.

**Comment by cwatss** [comment]
> To clarify the StringBuilder, it was used mainly a placeholder in case we wanted to increase the toString's capability, e.g. printing out a visual representation of the room with its doors.
> 
> Also, most of my code is based off the example program Tom shared in his assignment video.

Ok since were adding the doors and what not, this makes sense. I thought it was only to add the content of the room. I see where Tom used it in his example. Thank you 🙂👍

**Pull request merged**